target/scripts/helpers/ssl.sh

```diff
@@ -0,0 +1,409 @@
+#! /bin/bash
+
+function _setup_ssl
```

Comment:
I added the `_monitor_files_checksum` function and the `_extract_certs_from_acme` function to this file as well (and removed it from `helper-functions.sh`) in a first attempt to get rid of the `helper-functions.sh` script in the future (because we now have a proper `helpers/` directory). I added a note as well.